frontend/apps/pablo/tests/organisms/pool_liquidity_panel.test.tsx

```diff
@@ -4,7 +4,7 @@ import * as stories from "pablo-storybook/stories/organisms/pool_liquidity_panel
 
 const { Default } = composeStories(stories);
 
-test("renders PoolLiquidityPanel with default args", () => {
+test.skip("renders PoolLiquidityPanel with default args", () => {
```

Comment:
This is until Subsquid is deployed?

Reply:
And a bit further than that. needs all data.